packages/apollo-server/src/utils/exports.ts

```diff
@@ -0,0 +1,6 @@
+export * from 'graphql-tools';
```

Comment:
This file could use a comment. "Everything exported from this file is exported from top-level apollo-server and all of the variants."

Though huh, that's a pretty big interface — are we planning to maybe pare down graphql-tools at all ever?

Reply:
Can we be more explicit, in the name of defense?  Right now if `graphql-tool` exported something conflicting, stomping would occur.